### PR TITLE
wrk: cpp-beast simple-server - sample

### PR DESCRIPTION
**Pull request by kassane**
@renerocksai,

C++ server, [boost/beast forked to zig-pkg](https://github.com/kassane/beast) header-only.
No need install another deps, only zig.

```bash
========================================================================
                          cpp
========================================================================
Running 10s test @ http://127.0.0.1:8070
  4 threads and 400 connections
  Thread Stats   Avg      Stdev     Max   +/- Stdev
    Latency    12.56ms  749.24us  18.46ms   93.27%
    Req/Sec     7.96k   193.24     8.33k    74.75%
  Latency Distribution
     50%   12.44ms
     75%   12.81ms
     90%   13.10ms
     99%   15.06ms
  316709 requests in 10.04s, 30.20MB read
  Socket errors: connect 0, read 226240, write 90468, timeout 0
Requests/sec:  31536.32
Transfer/sec:      3.01MB
```

### CODE REVIEW

**Comment by kassane** [comment]
Regardless of the result who is better or not. The objective is to show that there are other possibilities of reference.

Same here:
- https://github.com/orhun/zig-http-benchmarks/pull/6

It won't do to criticize a tool without showing a usable example.

**Comment by renerocksai** [comment]
Thx for this. 

A personal note: I have come to dislike adding more benchmarks. They serve no purpose. To me, the existing benchmarks show enough: zap is fast enough, comparable with industry-standard go and some rust implementations; definitely faster than Python frameworks :).

People keep requesting to add more benchmarks. I have learned that the benchmarks only seem to serve ego purposes or are taken to start silly arguments.

Anyway, I am happy to "silently" put this in and when I have time, let you know the results from my bench test machine here; I propose to not add it to the bar chart in the README. To any sane person, it should be clear that C++ and C frameworks (like facil.io) are likely to be very fast, probably in the ballpark of zap / modern Rust / go implementations.

One more thing I'd like to ask: to keep things fair: can you change the response to be a static 17-byte string, please? That way we eliminate measuring file system access/caching, etc. All the other examples just respond with a static string, too.

**Review by renerocksai — CHANGES_REQUESTED** [review]
Please, to keep the wrk tests fair across frameworks, update the response to a static 17-byte string.

File: wrk/cpp/main.cpp [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            // Wait for a client to connect
+            tcp::socket socket(io_context);
+            acceptor.accept(socket);
+
+            // static 17-byte string
+            
+            // Read HTML content from a file (e.g., "index.html")
+            // std::string html_content = read_html_file("hello.html");
+            // or
+            std::string msg = "Hello from C++!!!";
```
Comment (renerocksai) [inline wrk/cpp/main.cpp:42]: small nitpick: you could move the definition of `msg` out of the loop. but that would mess the comment up. so, fine with me!

**Review by renerocksai — APPROVED** [review]

**Pull request merged**